Clients/Xamarin.Interactive.Client.Windows/Xamarin.Interactive.Client.Windows.csproj

```diff
@@ -51,17 +49,15 @@
     <PlatformTarget>AnyCPU</PlatformTarget>
     <DebugType>pdbonly</DebugType>
     <Optimize>true</Optimize>
-    <OutputPath>bin\Release$(ClientOutputPathSuffix)</OutputPath>
-    <IntermediateOutputPath>obj\Release$(ClientOutputPathSuffix)</IntermediateOutputPath>
-    <DefineConstants>WIN;TRACE;$(ClientDefineConstants)</DefineConstants>
+    <OutputPath>bin\Release\Workbooks</OutputPath>
+    <DefineConstants>WIN;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <PropertyGroup>
-    <ApplicationIcon Condition=" '$(ClientAssemblyName)' == 'Xamarin Inspector' ">xamarin-inspector.ico</ApplicationIcon>
-    <ApplicationIcon Condition=" '$(ClientAssemblyName)' == 'Xamarin Workbooks' ">xamarin-workbooks-client.ico</ApplicationIcon>
+    <ApplicationIcon>xamarin-workbooks-client.ico</ApplicationIcon>
```

Comment:
Placeholder comment for getting an issue filed about this.

Reply:
#229 

Do you get a phone call from GitHub as soon as a PR is opened?